### PR TITLE
Adding packages

### PR DESCRIPTION
**Pull request by dwarika-porter**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -390,6 +390,16 @@ prop-types@^15.6.2, prop-types@^15.7.2:
...
+  integrity sha512-Yf3gdUhQR5B2iyr6IHs6mXEdo/meiU6ZJvFXK3zLM/Tu3jJH1uc8T/Mzg19OQYITT9U9hC0HdHxtIU+LfnLXTg==
+
+react-ga@^3.3.0:
+  version "3.3.0"
+  resolved "https://registry.yarnpkg.com/react-ga/-/react-ga-3.3.0.tgz#c91f407198adcb3b49e2bc5c12b3fe460039b3ca"
+  integrity sha512-o8RScHj6Lb8cwy3GMrVH6NJvL+y0zpJvKtc0+wmH7Bt23rszJmnqEQxRbyrqUzk9DTJIHoP42bfO5rswC9SWBQ==
+
 react-is@^16.7.0, react-is@^16.8.1:
   version "16.13.1"
   resolved "https://registry.yarnpkg.com/react-is/-/react-is-16.13.1.tgz#789729a4dc36de2999dc156dd6c1d9c18cea56a4"
```
Comment (Prateek13727) [inline yarn.lock:405]: update V 1.1.2 in the readme as well with a short description

File: yarn.lock [hunk 2]
```diff
@@ -390,6 +390,16 @@ prop-types@^15.6.2, prop-types@^15.7.2:
     object-assign "^4.1.1"
     react-is "^16.8.1"
 
+react-dependent-script@^1.1.2:
```
Comment (Prateek13727) [inline yarn.lock:393]: are the versions for react-dependent-script and react-ga the latest

File: yarn.lock (hunk 1 above)
Comment (kd4001) [reply yarn.lock:405]: Sure

File: yarn.lock (hunk 2 above)
Comment (kd4001) [reply yarn.lock:393]: Yes

**Pull request merged**